.github/workflows/main.yml

```diff
@@ -195,7 +195,7 @@ jobs:
         --define sonar.projectName=libcanard
         --define sonar.projectKey=libcanard
         --define sonar.sources=libcanard
-        --define sonar.exclusions=libcanard/_canard_cavl.h
+        --define sonar.exclusions=lib/cavl2/cavl2.h
```

Comment:
It is likely that this is no longer needed since we moved it out of the main source dir. Please try removing it and see if sonar complains.